packages/xo-web/src/common/editable/index.js

```diff
@@ -121,7 +127,19 @@ class Editable extends Component {
 
       this.setState({ saving: true })
 
-      await saveValue(value)
+      const handlerParams = reduce(
```

Comment:
You can use `Object.keys(props).reduce()` to avoid an unnecessary dependency on `lodash`.